mesonbuild/minit.py

```diff
@@ -22,7 +22,9 @@
 import os
 import re
 from glob import glob
-from mesonbuild import mesonlib
+from . import mlog
+from .mesonlib import setup_vsenv
```

Comment:
We can just use `mesonlib.setup_vsenv` instead of importing this import twice, probably.